splinter/include/splinter_ros/splinter1d.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef SPLINTER_ROS__SPLINTER1D_HPP_
```

Comment:
not sure what license to put here (or .cpp or on splinter2d.hpp/.cpp)... I made a wrapper that completely hides splinter and their bundled eigen

Reply:
the whole point of the separate package was to use their license 🤣

Reply:
I copy their license file into the top level of the package during build/install

Reply:
For maximum compatibility, probably just stick to MPL-2.0.

If there are other issues with that, then you can license your newly-written code however you want.